### ISSUE TITLE
allow show Tarball Contents info like npm 6.x do

### ISSUE THREAD

**Issue by bluelovers** [yarnpkg/yarn#6456]
**Do you want to request a *feature* or report a *bug*?**

at npm 6.x when `npm publish` will show Tarball Contents info

hope yarn can have option flag for this too

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**Please mention your node.js, yarn and operating system version.**

**Comment by arcanis** [comment]
Agree, it would be a nice feature! Would you like to open a PR and implement it?

**Comment by bluelovers** [comment]
@arcanis if tell me where is source code about it in yarn, i can try  implement it

**Comment by arcanis** [comment]
The packing logic is here: https://github.com/yarnpkg/yarn/blob/master/src/cli/commands/pack.js
